psftransltator.py

```diff
@@ -1,13 +1,17 @@
+#!/usr/bin/env python
 
 from logging import error, log
-
+import sys
 
 def loadProgram(filename):
     f = open(filename)
     lines = f.readlines()
     return lines
 
-lines = loadProgram("test.pst")
+if(len(sys.argv) > 1):
+    lines = loadProgram(sys.argv[1])
+else:
+    lines = loadProgram("test.pst")
```

Comment:
This means you can type `./psftranslator.py boo.pst` and it will run `boo.pst` instead of `test.pst`!